xml/vk.xml

```diff
@@ -14163,10 +14178,17 @@ typedef void <name>CAMetalLayer</name>;
                 <enum value="&quot;VK_NV_extension_351&quot;"               name="VK_NV_EXTENSION_351_EXTENSION_NAME"/>
             </require>
         </extension>
-        <extension name="VK_EXT_extension_352" number="352" author="EXT" contact="Joshua Ashton @Joshua-Ashton" supported="disabled">
+        <extension name="VK_VALVE_mutable_descriptor_type" number="352" type="device" supported="vulkan" author="VALVE" contact="Joshua Ashton @Joshua-Ashton,Hans-Kristian Arntzen @HansKristian-Work">
```

Comment:
Should we add a specialuse="d3demulation"? Or do we believe this is general enough to be recommended for use by general vulkan apps?

Reply:
Should be d3demulation, yes.